first-day/intro/index.html

```diff
@@ -182,6 +182,28 @@
 
 <p>4:45-5:30 - Recap on day with buddies and any questions about the Homework Task.</p>
 
+<h3>Week 3:</h3>
+
+<p>9:00-9:30 - Sign in and catch up with buddy - Canteen</p>
+
+<p>9:30-10:30 - Stand-up and recap session from last week (Git, Marshmallow exercise, Wireframing) - Laura + Jameel </p>
```

Comment:
As per our discussion in the meeting lets bring down the recap session to 15 minutes. We will run it as a standup session. I.e. I did this since last week, I'll be doing this today.

Reply:
Think 15 mins is enough though? 

Am thinking with the content building up it might be useful to recap on both the previous weeks stuff, and also just check things like git/ etc that might have been the week before that, just to ensure no one is lagging - wdyt?